plugins/citationLookup/crossref/locale/fi_FI/locale.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE locale SYSTEM "../../../../lib/pkp/dtd/locale.dtd">
```

Comment:
`<!DOCTYPE locale SYSTEM "../../../../../dtd/locale.dtd">`

Reply:
ok, will fix these. I used the translation plugin, so it has to be generating those.